src/Makefile

```diff
@@ -274,6 +274,15 @@ ifeq ($(MALLOC),jemalloc)
 	FINAL_LIBS := ../deps/jemalloc/lib/libjemalloc.a $(FINAL_LIBS)
 endif
 
+# only Linux has IO_URING support
+ifeq ($(uname_S),Linux)
+	USE_IO_URING=yes
```

Comment:
Linux version > 5.1 checker is not very accurate, for the kernel version older than 5.1, the kernel could also support io_uring by manually backoff the io_uring patch. If the kernel really don't support io_uring yet, when Redis start, it will be checked in `initIOUring` by `io_uring_queue_init_params(3)`, it will return a -error on failure.